install/docker-compose/docker-compose.yml

```diff
@@ -18,10 +23,20 @@ services:
     volumes:
       - "./keycloak-realm/microcks-realm-sample.json:/opt/keycloak/data/import/microcks-realm.json"
     command: ["start-dev", "--hostname=http://localhost:18080", "--import-realm"]
+    healthcheck:
+      test: ["CMD", "sh", "-c", "echo > /dev/tcp/localhost/8080"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
 
   postman:
     image: quay.io/microcks/microcks-postman-runtime:0.6.0
     container_name: microcks-postman-runtime
+    healthcheck:
+      test: ["CMD", "sh", "-c", "echo > /dev/tcp/localhost/3000"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
```

Comment:
We have integrated probes in Microcks Postman Runtime that are already used on the Kubernetes deployment. Ideally, we must converge and use the same ones.